nucypher/utilities/prometheus/collector.py

```diff
@@ -117,7 +119,11 @@ def _collect_internal(self) -> None:
 
         self.metrics["learning_status"].state('running' if self.ursula._learning_task.running else 'stopped')
         self.metrics["known_nodes_gauge"].set(len(self.ursula.known_nodes))
-        self.metrics["work_orders_gauge"].set(len(self.ursula.work_orders()))
+        try:
```

Comment:
IMO, this belongs in a method on Ursula.

Reply:
I'd like to make this a query in `datastore.queries` instead. I'll do that in a follow-up PR. :+1: